src/Import/viewpoint/import_viewpoint.m

```diff
@@ -49,6 +49,13 @@
     [channels, marker, chan_info] = parse_messages(messages, channels, marker, chan_info, file_info.eyesObserved);
 
     sess_beg_indices = [find(contains(marker, '+')); size(dataraw, 1) + 1];
+
+    % After setting the beginning index we delete all ',','+','=' form markers array
+    NotMarkers = [',','+','=']; 
+    for nmkr=1:size(NotMarkers,2)
+       marker = strrep(marker,NotMarkers(nmkr),'');
+    end
+

```

Comment:
1. I believe that this filtering should be implemented in `pspm_get_viewpoint` function, not in `import_viewpoint`. There are two reasons for this:
  * It is highly likely that doing this kind of filtering in `import_viewpoint` itself will require modifications to other sections, as well. Therefore, it will require more changes than this.
  * It is against the general rationale behind `import` and `get` function separation. `import` functions are intended to return whatever is in the datafile in some sort of structured format, whereas `get` functions are intended to process this and make it compatible with PsPM functions.
2. Note that while filtering the markers according to name, you should also get rid of corresponding timesteps.
3. We need to do the filtering a bit more strictly since these characters may also appear in proper marker names such as `CS+`. For filtering the name, I would suggest removing the markers whose names consist of only `,+=` characters.